src/modules/src/Eryph.Modules.ComputeApi/Endpoints/V1/ProjectMembers/ProjectMemberRequest.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Eryph.Modules.AspNetCore.ApiProvider.Model;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Eryph.Modules.ComputeApi.Endpoints.V1.ProjectMembers;
+
+public class ProjectMemberRequest : SingleEntityRequest
+{
+    [FromRoute(Name = "project")] public virtual string Project { get; set; }
```

Comment:
I have tried to catch all cases where project is used instead of projectId. For nullable guid your are right, but I remember that I had some issues with inconsistent behavoir for validations. However this would be better a topic for this #159 .

Reply:
fixed the guids

Reply:
A, now I see your point, you mean the property name. Ok, will update that as well.